app/bootstrap.php

```diff
@@ -4,6 +4,32 @@
  *
  * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
  */
-require __DIR__ . '/../vendor/autoload.php';
+
+$files = [
+    __DIR__ . '/../../../autoload.php',
+    __DIR__ . '/../vendor/autoload.php',
+    __DIR__ . '/vendor/autoload.php'
+];
+
+foreach ($files as $file) {
+    if (file_exists($file)) {
+        define('INFECTION_COMPOSER_INSTALL', $file);
+
+        break;
+    }
+}
+unset($file);
```

Comment:
the variable has been defined just above with the `foreach()`, it doesn't get unset as you exit the loop. Not sure if it's worth to remove unset though, or if you do I would unset `files` as well or maybe do that in a closure to avoid having to unset any variable